charts/spire/charts/spire-server/values.yaml

```
@@ -707,6 +707,8 @@ tornjak:
     # requests:
     #   cpu: 100m
     #   memory: 128Mi
+  ## @param tornjak.securityContext [object] Security Context to use
```

```suggestion

  ## @param tornjak.securityContext [object] Security Context to use
```

Maybe white line in between for readability